tools/dep_updaters/update-googletest.sh

```diff
@@ -0,0 +1,96 @@
+#!/bin/sh
+set -e
+# Shell script to update GoogleTest in the source tree to the most recent version.
+# GoogleTest follows the Abseil Live at Head philosophy and rarely creates tags
+# or GitHub releases, so instead, we use the latest commit on the main branch.
+
+BASE_DIR=$(cd "$(dirname "$0")/../.." && pwd)
+DEPS_DIR="$BASE_DIR/deps"
+
+[ -z "$NODE" ] && NODE="$BASE_DIR/out/Release/node"
```

Comment:
I dont think $NODE is used anywhere, I think it can be removed

Reply:
Yes, removing it will not make any further difference.